src/main/java/core/util/StringUtils.java

```diff
@@ -79,8 +79,10 @@ public static String getResultString(int homeGoals, int awayGoals) {
 		if (awayGoals < 10) {
 			buffer.append(" ");
 		}
```

Comment:
from what I see here it looks the score should be correctly displayed (no extra space) maybe the screenshot I look at to make my general remark was not based on that code, if that is the case you can discard my comment

Reply:
(Side note: some day we may want to look into l10n'ing this, as AFAIK using colons to separate scores is a germanophone thing.  For example in English it is more common to use a dash, e.g. “ 2 – 1”.)

Reply:
@akasolace the intention of the space code was the alignment of the colons in case of results with more than 9 goals. this we loose with the result extensions. if there is no other trick, we could change it by a space at the beginning of the result string only (if home team has less than 10 goals) and left align the cells instead of centering them.
@tychobrailleur language text for the separation sign is a good idea

Reply:
+1 for  l10n'ing the separation